astropy/time/core.py

```diff
@@ -2157,6 +2157,10 @@ def to(self, *args, **kwargs):
         return u.Quantity(self._time.jd1 + self._time.jd2,
                           u.day).to(*args, **kwargs)
 
+    def to_value(self, *args, **kwargs):
```

Comment:
In general I prefer to have the actual function signature here instead of just delegating to whatever `Quantity.to_value()` takes.  While the current approach makes this method more future-proof for `Quantity.to_value` API changes (probably not likely), it makes it more opaque for the user and typing `dt.to_value?` will give no useful help.

At the very least, need a docstring that provides information that this is calling `Quantity.to_value`, but preferably also shows the `unit` and `equivalencies` parameters.  BTW, are there any equivalencies relevant for time delta units?

Reply:
I just copied from `.to` above, but I agree with your sentiment and will update it. (And, no, don't think there are astropy-defined equivalencies that are relevant, but of course a user might put in anything...)

Reply:
Of the top of my head, I can think of time / frequency equivalency. Transforming a timedelta into a frequency.

Reply:
@taldcroft - I now updated the `TimeDelta.to_value` and `.to` methods with proper docstrings.